tests/test_utils.py

```diff
@@ -135,3 +137,23 @@ def test_num_parameters():
     assert num_parameters(model) == 6
     assert num_parameters(model, requires_grad=True) == 4
     assert num_parameters(model, requires_grad=False) == 2
+
+
+@RunIf(min_cuda_gpus=1)
+@pytest.mark.parametrize("mode", ["nf4", "nf4-dq", "fp4", "fp4-dq", "int8", "int8-training"])
+@pytest.mark.skip("To be fixed")
```

Comment:
Here's your homework @Andrei-Aksionov :books:

Reply:
Thanks a lot, of course 😆.
But, teacher, I don't understand the task.

That problematic PR #715 updated the way to count number of parameters (to reflect changes in BNB 0.41.2), but introduced another problem #734.
So this test should pass without any problems.